java/src/Ice/src/main/java/com/zeroc/IceSSL/SSLEngine.java

```diff
@@ -624,156 +433,23 @@ else if(properties.getPropertyAsInt("IceSSL.UsePlatformCAs") <= 0)
                     assert(trustManagers != null);
                 }
 
-                //
-                // Create a certificate path validator to build the full
-                // certificate chain of the peer certificate. NOTE: this must
-                // be done before wrapping the trust manager since our wrappers
-                // return an empty list of accepted issuers.
-                //
-                _validator = CertPathValidator.getInstance("PKIX");
```

Comment:
We no longer have to build a custom CertPathValidator, we use the context trust manager created from the configured CAs. This didn't work previously because of the custom `X509TrustManagerI`.